src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,64 @@
+package duke;
```

Comment:
I may advise against the use of this class because it seems to me that this class is emulating the Arraylist class!